pinot-hadoop/src/main/java/org/apache/pinot/hadoop/job/SegmentPreprocessingJob.java

```diff
@@ -218,34 +431,20 @@ private void setMaxNumRecordsConfigIfSpecified(Job job) {
    * @return Input schema
    * @throws IOException exception when accessing to IO
    */
-  private static Schema getSchema(Path inputPathDir)
```

Comment:
We don't event need the path as the param, right?

Reply:
we do, we list status of this input path

Reply:
no, we don't need to